01-weatherpattern.Rmd

```diff
@@ -1,11 +1,584 @@
+
+When focusing only on the frequency of HB CTs, without considering their duration, it is important to highlight that the duration of a HB CT significantly influences the amplitude of weather extremes. This underscores the need to integrate both frequency and duration for a more comprehensive understanding of weather patterns. Duration of a HB CT impacts the amplitude of a weather extreme.
+
+To further evaluate the uncertainties in frequency changes, it would be beneficial to combine multi-model and single-model ensembles under different forcing scenarios. This approach could help assess the uncertainties arising from various climate models and the assumptions in different forcing scenarios.
+
+For future studies, we recommend quantifying human-level errors in labeling to understand better the impacts of manual classification biases. 
+Additionally, models that explicitly capture temporal continuity, such as Deep Hidden Markov Models [@Yu2015] or temporal-aware CNN architectures (e.g., ConvLSTM [@Moishin2021]), 
+could enhance prediction accuracy by directly modeling the persistence characteristic inherent in circulation patterns [@Mittermeier2022].
+
+## Appendix
+
+### Statement
+
+The implementation was carried out using Python version 3.9.18.
+
+### Figures
+
+Synoptic patterns of the 29 HB CTs using ERA20C (Figure \@ref(fig:ERA20C))
+
+```{r ERA20C,fig.cap = 'Synoptic patterns of the 29 HB CTs created using ERA-20C data averaged over the period 1900 -1980, showing SLP on the left and z500 on the right side [@Mittermeier2022].', cache=FALSE, out.width="70%", fig.align="center", echo=FALSE, eval = TRUE}
+knitr::include_graphics('work/01-weatherpattern/figures/paper/circulation_types.png')
+```
+
+HB CTs (Figure \@ref(fig:cts))
+
+```{r cts,fig.cap = 'HB CTs', cache=FALSE, out.width="70%", fig.align="center", echo=FALSE, eval = TRUE}
+knitr::include_graphics('work/01-weatherpattern/figures/paper/cts.png')
+```
+
+F1 score comparison table (Figure \@ref(fig:f1score))
+
+```{r f1score,fig.cap = 'Macro F1', cache=FALSE, out.width="70%", fig.align="center", echo=FALSE, eval = TRUE}
+knitr::include_graphics('work/01-weatherpattern/figures/f1.png')
+```
+
+Signature plots of paper - SLP (Figure \@ref(fig:sigpaper))
+
+```{r sigpaper,fig.cap = "Signature plots of four selected circulation patterns at slp. Column 1: labels showing the indicated HB CT, Column 2: deep ensemble predictions showing the indicated HB CT, Column 3: signature pattern, when the deep ensemble predicts the indicated HB CT while labels state differently, Column 4: labels stating the indicated HB CT while deep ensemble predicts differently. The RMSE values are calculated by comparing the respective signature plot to the signature plot of the labels (column 1). The four HB CTs are chosen as positive (green) and negative (red) examples [@Mittermeier2022].", cache=FALSE, out.width="70%", fig.align="center", echo=FALSE, eval = TRUE}
```

Comment:
> ...examples [@Mittermeier2022]

This won't work in the pdf rendering. Here is a minimal example of how to make cites in figure captions work:

(ref:captionScatter) @genest2013
```
```{r scatterClaudia, echo=FALSE, message=FALSE, warning=FALSE, out.width="60%", fig.align='center', fig.cap="Scatter plot of the pseudo-observations \\((u_1, v_1) \\ldots, (u_n, v_n)\\) from \\(C\\), where \\(u_i\\) is calculated from data on the Inn and \\(v_i\\) is obtained from data on the Danube (\\(\\forall i \\in \\{1,\\ldots, n\\}\\)). Adapted from (ref:captionScatter), p. 93."}
include_graphics("./work/04-archimax/figures/Plot_Inn_Danube_raw_adjusted_v02.png")
```
```